docs/units/equivalencies.rst

```diff
@@ -25,30 +25,28 @@ piece of code needs the same equivalencies, one can set them for a
 Built-in equivalencies
 ======================
 
-Parallax Units
---------------
+How to Convert Parallax to Distance
+-----------------------------------
+
+The length unit *parsec* is defined such that a star one parsec away will exhibit a 1-arcsecond parallax. (Think of it as a contraction between *parallax* and *arcsecond*.)
 
-:func:`~astropy.units.equivalencies.parallax` is a function that returns an
-equivalency list to handle conversions between angles and length.
+The :func:`~astropy.units.equivalencies.parallax` function handles conversions between parallax angles and length.
 
-Length and angles are not normally convertible, so
+In general, you should not be able to change units of length into angles or vice versa, so
 :meth:`~astropy.units.core.UnitBase.to` raises an exception::
 
   >>> from astropy import units as u
-  >>> (8.0 * u.arcsec).to(u.parsec)  # doctest: +IGNORE_EXCEPTION_DETAIL
+  >>> (1.0 * u.arcsec).to(u.parsec)  # doctest: +IGNORE_EXCEPTION_DETAIL
   Traceback (most recent call last):
     ...
   UnitConversionError: 'arcsec' (angle) and 'pc' (length) are not convertible
 
-However, when passing the result of
-:func:`~astropy.units.equivalencies.parallax` as the third argument to the
-:meth:`~astropy.units.core.UnitBase.to` method, angles can be converted
-into units of length (and vice versa).
+To trigger the conversion between parallax angle and distance, provide 
+:func:`~astropy.units.equivalencies.parallax` as the third argument (``equivalencies=``) to the
```

Comment:
I think the original "third" wording is misleading, as users do not really see `self` being counted when they look at the stuff passed into parenthesis. However, if you look at the API and count `self`, then it is the third. 🤔 
```suggestion
:func:`~astropy.units.equivalencies.parallax` as the optional keyword argument (``equivalencies=``) to the
```

Reply:
Should I change it back, or not?  I took out "third" precisely for that reason. We are prepping for the example gallery, so it is likely that users will see this example separate from the API docs. For that reason, the probability for confusion is high.

Reply:
Oh oops, I thought I had changed it but apparently did not. Done!

Reply:
I had gone back and forth on that in my mind! So much so that I thought your suggestion was what I had actually written.